lib/v2/wtu/job.js

```diff
@@ -0,0 +1,77 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { URL } = require('url');
```

Comment:
```suggestion
```
No need to require global object.
docs: https://nodejs.org/api/url.html#class-url